frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -24,6 +24,7 @@ const WARNING_TYPE_TO_DESCRIPTION = {
     ignored_invalid_timestamp: 'Ignored an invalid timestamp, event was still ingested',
     event_timestamp_in_future: 'An event was sent more than 23 hours in the future',
     ingestion_capacity_overflow: 'Event ingestion has overflowed capacity',
+    message_size_too_large: 'Discarded event exceeding 1MB limit',
```

Comment:
Corresponding documentation change is over here: PostHog/posthog.com#7074

(I assume the description and title cases must match and that the documentation site autogenerates anchors?)